content/kotlin/concepts/strings/strings.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'Strings'
+Description: 'Strings are immutable objects that are sequences of characters contained within a pair of double quotes.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Data Types'
+  - 'Characters'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+**Strings** are immutable objects that represents a sequence of characters contained within double quotes (`""`).
+
+## Syntax
+
+```pseudo
+val stringName: String = "string value"
+```
+
+In Kotlin, string values are always defined as instances of the `String` class.
```

Comment:
```suggestion
In Kotlin, string values are always defined as instances of the `String` class.

### String Templates

Templates can also be used to dynamically produce string values. Dynamic values don't have to start as strings, and can either be directly referenced with a dollar sign (`$`) or evaluated as a string with additional curly braces (`{}`).

```

Reply:
After a second look, I'd suggest placing the information about string templates directly below the end of the "Syntax" section and changing to a lv. 3 header.